cloudquery/sdk/scalar/json.py

```diff
@@ -18,7 +18,7 @@ def set(self, value: any):
         if value is None:
             return
 
-        if type(value) == str or type(value) == bytes:
+        if isinstance(value, (str, bytes)):
```

Comment:
connected to the comment about List scalar: here's a valid usage of `isinstance`, as it works as `one of`